release/benchmarks/distributed/test_many_actors.py

```diff
@@ -29,11 +29,9 @@ def foo(self):
         for _ in tqdm.trange(MAX_ACTORS_IN_CLUSTER, desc="Launching actors")
     ]
 
-    not_ready = [actor.foo.remote() for actor in actors]
-
-    for _ in tqdm.trange(len(actors)):
-        ready, not_ready = ray.wait(not_ready)
-        assert ray.get(*ready) is None
+    done = ray.get([actor.foo.remote() for actor in actors])
```

Comment:
Hmm I still don't understand how this can have almost 2X regression? It is only 5000 object refs...